utils/kafka-test-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/EmbeddedKafkaServer.scala

```diff
@@ -111,11 +118,17 @@ object EmbeddedKafkaServer {
 
 class EmbeddedKafkaServer(
     zooKeeperServerOpt: Option[(NIOServerCnxnFactory, ZooKeeperServer, File)],
-    val kafkaServer: Server,
+    createKafkaServer: () => Server,
     val kafkaAddress: String,
     tempDir: File
 ) extends LazyLogging {
 
+  var kafkaServer: Server = createKafkaServer()
```

Comment:
why do we have to expose the underlying server? let's keep it as internal

Reply:
It's internal now